gh-pages/_index.html

```diff
@@ -0,0 +1,472 @@
+        Dropdown Grid Field
+      </h4>
+      <p>
+        A dropdown field that displays the available choices in a grid.
+      </p>
+    </div>
+  </a>
+  <a class="card" href="plugins/field-multilineinput/test/index.html">
+    <img srcset="media/multiline-field.png 2x" alt="">
+    <div>
+      <h4>
+        Multi-Line Text Field
+      </h4>
+      <p>
+        A field that lets users enter multiple lines of text.
+      </p>
+    </div>
+  </a>      
+  <a class="card vertical" href="plugins/field-dependent-dropdown/test/index.html">
+    <video autoplay playsinline muted loop width="375px">
+      <source src="media/dependent-dropdown-field.mp4" type="video/mp4">
+    </video>
+    <div>
+      <h4>
+        Dependent Dropdown Field
+      </h4>
+      <p>
+        A dropdown field whose available values depend on the chosen value in a separate dropdown field.
+      </p>
+    </div>
+  </a>
+</div>
+
+<h3 id="blocks">
+  Blocks
+</h3>
+<p>
+  These plugins provide new types of blocks or modify the behavior of existing ones. Learn more about <a href="https://developers.google.com/blockly/guides/create-custom-blocks/overview">creating custom blocks</a>.
+</p>
+<div class="grid">
+  <a class="card" href="plugins/block-dynamic-connection/test/index.html">
+    <video autoplay playsinline muted loop width="229px">
+      <source src="media/dynamic-connections.mp4" type="video/mp4">
+    </video>
+    <div>
+      <h4>
+        Dynamic Connection Blocks
+      </h4>
+      <p>
+        A plugin that provides custom text, list and conditional blocks that dynamically add and remove inputs as the user interacts with them.
+      </p>
+    </div>
+  </a>
+  <a class="card" href="plugins/block-plus-minus/test/index.html">
+    <video autoplay playsinline muted loop width="211px">
+      <source src="media/plus-minus.mp4" type="video/mp4">
+    </video>
+    <div>
+      <h4>
+        +/- Mutator Blocks
+      </h4>
+      <p>
+        Custom logic, list, text, and function blocks that use + and - controls to add and remove inputs instead of the normal mutator interface.
+      </p>
+    </div>
+  </a>  
+  <a class="card" href="plugins/renderer-inline-row-separators/test/index.html">
+    <img srcset="media/inline-inputs.png 2x">
+    <div>
+      <h4>
+        Inline Input Blocks
+      </h4>
+      <p>
+        A custom renderer and example blocks that treat dummy inputs between value inputs as line breaks, rendering the value inputs on separate lines inside of the block.
+      </p>
+    </div>
+  </a>
+  <a class="card" href="plugins/block-shareable-procedures/test/index.html">
+    <img srcset="media/procedure-blocks.png 2x">
+    <div>
+      <h4>
+        Procedure Blocks
+      </h4>
+      <p>
+        These blocks allow users to define and call procedures. Although the core Blockly library includes procedure blocks that behave identically for end users, developers that don't need backwards compatibility should use these instead.
```

Comment:
@BeksOmega this phrasing sounds like the two sets of blocks are the same. Do you have suggestions on short ways to explain the differences?

Reply:
How 'bouts:

"These blocks allow users to define and call procedures. These have more functionality than the equivalent blocks in core, which allows them to be shared between workspaces. The Blockly team recommends that developers use these instead."

It's not exactly shorter but I think it does explain the difference?

Reply:
Done!